.github/workflows/build.yml

```diff
@@ -127,6 +127,9 @@ jobs:
           gradle-cache-read-only: ${{ !inputs.is-trunk }}
           gradle-cache-write-only: ${{ inputs.is-trunk }}
           develocity-access-key: ${{ secrets.DEVELOCITY_ACCESS_KEY }}
+      - name: Verify license file
+        if: always()
```

Comment:
`always()` means this will run even if a previous step failed. I think we don't need this in this step, WDYT?

Reply:
@m1a2st  Thanks for the code review, and you are right, I have fixed it, PTAL